### ISSUE TITLE
Open in Editor menu button is missing aria label

### ISSUE THREAD

**Issue by timtmok** [posit-dev/positron#5894]
#### Positron and OS details:

2025.01

## Describe the issue:
The dropdown button is missing an aria label. A screen reader won't describe the element at all when it has focus.

![Image](https://github.com/user-attachments/assets/f02985d5-aab5-4909-9e49-9a211c16cda0)
![Image](https://github.com/user-attachments/assets/6b022b3b-1d0f-4a69-9037-2804fdb8a861)

## Steps to reproduce the issue:

1. Activate a screen reader
2. Render a plot
3. Focus on the dropdown arrow on the Open in Editor button in the Plots view action bar

## Expected or desired behavior:
A screen reader should describe that the element will open a menu to select where the plot will be opened.

**Comment by timtmok** [comment]
Introduced from #5733

**Comment by testlabauto** [comment]
### Verified Fixed
    Positron Version(s) : 2025.02.0-127
    OS Version          : OSX

### Test scenario(s) 

Label is now present!

Link(s) to TestRail test cases run or created:
